test/ffmpeg-qsv/encode/av1.py

```diff
@@ -0,0 +1,54 @@
+###
+### Copyright (C) 2019-2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.ffmpeg.qsv.util import *
+from ....lib.ffmpeg.qsv.encoder import EncoderTest
+
+spec = load_test_spec("av1", "encode", "8bit")
+
+#@slash.requires(*have_ffmpeg_encoder("av1_qsv"))
+@slash.requires(*have_ffmpeg_decoder("av1_qsv"))
+class AV1EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec     = "av1-8",
+      ffencoder = "av1_qsv",
+      ffdecoder = "av1_qsv",
+    )
+
+  def get_file_ext(self):
+    return "ivf"
+
+@slash.requires(*platform.have_caps("vdenc", "av1_8"))
+class AV1EncoderLPTest(AV1EncoderBaseTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      caps      = platform.get_caps("vdenc", "av1_8"),
+      lowpower  = 1,
+    )
+
+class cqp_lp(AV1EncoderLPTest):
+  def init(self, tspec, case, gop, bframes, tile_cols, tile_rows,qp, quality, profile):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = gop,
+      bframes   = bframes,
+      qp        = qp,
+      rcmode    = "cqp",
+      quality   = quality,
+      #profile   = profile,
+      tile_rows = tile_rows,
+      tile_cols = tile_cols,
+    )
+
+  @parametrize_with_unused(*gen_av1_cqp_lp_parameters(spec), ['quality'])
```

Comment:
Does libva support "quality" for av1e?  If not, then remove "quality" from parameters because no plugin will ever need it.  If so, then does the ffmpeg plugin support it?